src/Orc.Controls.Examples/ViewModels/DateRangePickerViewModel.cs

```diff
@@ -0,0 +1,77 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DateRangePickerViewModel.cs" company="WildGums">
+//   Copyright (c) 2008 - 2017 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Orc.Controls.Examples.ViewModels
+{
+    using System;
+    using System.ComponentModel;
+    using System.Globalization;
+    using System.Threading.Tasks;
+    using Catel.Collections;
+    using Catel.Data;
+    using Catel.MVVM;
+    using Catel.Threading;
+    using Models;
+    using System.Collections.ObjectModel;
+
+    public class DateRangePickerViewModel : ViewModelBase
+    {
+        #region Constructors
+        public DateRangePickerViewModel()
+        {
+
+        }
+        #endregion
+
+        #region Properties
+        public FastObservableCollection<DateRange> Ranges { get; set; }
```

Comment:
Yes, they must be set from outside, date ranges are not defined inside viewmodel, but can be assigned from outside. That way we allow to customize it.